### PR TITLE
Fix rubocop

### PR DESCRIPTION
**Pull request by kassio**

### CODE REVIEW

**Comment by xiaoshihou514** [comment]
Thanks for fixing this! Could you maybe write a test to verify it's parsing diagnostics correctly? There are insturctions for doing so in CONTRIBUTING.md

**Comment by kassio** [comment]
I tried to add tests, but this is a bit tricky as we'd need to install bundler, run `bundle install` before really executing `bundle exec rubocop...`

Is it possible to configure a linter/formatter partially? If so we could change the `cmd`/`args` to not use bundler by default. 🤷

File: README.md [hunk 1]
```diff
@@ -26,21 +26,22 @@
 - [ ] [rubocop](https://github.com/rubocop/rubocop)
 - [x] [rustfmt](https://github.com/rust-lang/rustfmt)
 - [ ] [shfmt](https://github.com/mvdan/sh)
+- [x] [sql-formatter](https://github.com/sql-formatter-org/sql-formatter)
```
Comment (kassio) [inline README.md:29]: Sorted the list

**Comment by xiaoshihou514** [comment]
> I tried to add tests, but this is a bit tricky as we'd need to install bundler, run `bundle install` before really executing `bundle exec rubocop...`

You can just put the table in the config, change the cmd, test that it's ok, and put the string back. That _should_ ensure the config at least works.

> Is it possible to configure a linter/formatter partially? If so we could change the `cmd`/`args` to not use bundler by default. 🤷

One could require the `parse` field from the repo I guess🤔, but it's up to you, if you think more people use rubocop not installed by bundler feel free to change that.

**Comment by kassio** [comment]
I decided to not use guard.nvim anymore. Thanks for the plugin, but it's not for me.

**Pull request closed without merging**